### PR TITLE
Fix up anaconda and zeppelin env paths

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: ansible/files/bootstrap/telemetry.sh [hunk 1]
```diff
@@ -148,20 +148,19 @@ export ANACONDA_PATH={{telemetry_analysis_anaconda_path}}
...
-
-rm /mnt/$ANACONDA_SCRIPT
 rm $PIP_REQUIREMENTS_FILE
 
 $ANACONDA_PATH/bin/conda create -n zeppelin python=3.6 cairo pillow -y -q
-source activate zeppelin
+source $ANACONDA_PATH/bin/activate zeppelin
 aws s3 cp $TELEMETRY_CONF_BUCKET/bootstrap/python3-requirements.txt $PIP_REQUIREMENTS_FILE
-pip install -r $PIP_REQUIREMENTS_FILE
+$CONDA_PREFIX/bin/pip install -r $PIP_REQUIREMENTS_FILE
```
Comment (comloo) [inline ansible/files/bootstrap/telemetry.sh]: Where is `CONDA_PREFIX` being defined?

File: ansible/files/steps/batch.sh [hunk 2]
```diff
@@ -89,17 +89,17 @@ else
     fi
     if [ $EXTENSION = "json" ]; then
         # Executes Zeppelin notebook
-        ZEPPELIN_PATH=/mnt/anaconda2/envs/zeppelin
-        source activate zeppelin
-        ${ZEPPELIN_PATH}/bin/zeppelin-execute -i ../${NOTEBOOK_NAME} -o ./${NOTEBOOK_NAME}
+        ANACONDA_PATH={{telemetry_analysis_anaconda_path}}
+        source $ANACONDA_PATH/bin/activate zeppelin
+        $CONDA_PREFIX/bin/zeppelin-execute -i ../${NOTEBOOK_NAME} -o ./${NOTEBOOK_NAME}
```
Comment (comloo) [inline ansible/files/steps/batch.sh]: Wondering if we change the syntax to `${CONDA_PREFIX}` to match how `${NOTEBOOK_NAME}` is being called, or vice versa.

File: ansible/files/bootstrap/telemetry.sh (hunk 1 above)
Comment (robhudson) [inline ansible/files/bootstrap/telemetry.sh]: It gets added to the env when you call activate.

File: ansible/files/steps/batch.sh (hunk 2 above)
Comment (robhudson) [inline ansible/files/steps/batch.sh]: Yeah, I considered that. No reason for the brackets.

**Pull request merged**